docs/reference/configuration/configuration.md

```diff
@@ -7,7 +7,23 @@ The Arm Mbed OS configuration system, a part of the Arm Mbed OS build tools, cus
 - The receive buffer size of a serial communication library.
 - The flash and RAM memory size of an Mbed target.
 
-The Arm Mbed OS configuration system gathers and interprets the configuration defined in the target in its [target configuration](../reference/adding-and-configuring-targets.html), all `mbed_lib.json` files and the `mbed_app.json` file. The configuration system creates a single header file, `mbed_config.h`, that contains all of the defined configuration parameters converted into C preprocessor macros. `mbed compile` places `mbed_config.h` in the build directory, and `mbed export` places it in the application root. `mbed compile` runs the Mbed configuration system before invoking the compiler, and `mbed export` runs the configuration system before creating project files.
+The Arm Mbed OS configuration system gathers and interprets the configuration defined in the target in its [target configuration](../reference/adding-and-configuring-targets.html), all `mbed_lib.json` files and the `mbed_app.json` file. The configuration system creates a single header file, `mbed_config.h`, that contains all of the defined configuration parameters converted into C preprocessor macros. 
+
+Here is a sample JSON file:
+
+```JSON
+"target_overrides": {
+    "*": {
+        "cellular.random_max_start_delay": "100"
+    },
+    "K64F": {
+        "cellular.use-apn-lookup": false,
+        "platform.stdio-baud-rate": 9600
+    }
+}
+```
+
+If you use the example JSON snippet above, you can see the macro `MBED_CONF_CELLULAR_RANDOM_MAX_START_DELAY` in `mbed_config.h`. This macro is created for all targets and is set to `100`, whereas macro `MBED_CONF_PLATFORM_STDIO_BAUD_RATE` is set to `9600` only for the K64F. `mbed compile` places `mbed_config.h` in the build directory, and `mbed export` places it in the application root. `mbed compile` runs the Mbed configuration system before invoking the compiler, and `mbed export` runs the configuration system before creating project files.
```

Comment:
I don't understand the bit about the K64F. We don't mention it before this line. Could you please clarify?

Reply:
So if we take a look at the snippet, there are parameters that are defined under "*", and others that are defined for specific target (K64F in this case). The intent is to make sure that users understand the scope of a macro value based on the targets its defined for. Would it make the documentation clearer if I mention here that "*" is a wildcard that applies to all targets?

Reply:
I think it would, but only if our developers aren't expected to know that.

Reply:
If they are, then please ignore.

Reply:
That stuff is very basic, and it is mentioned later in this documentation.